.azure/modules/functionApp/slackNotifier.bicep

```
@@ -155,6 +155,30 @@ resource notifyDevTeam 'Microsoft.Insights/actionGroups@2023-01-01' = {
   tags: tags
 }
 
+var query = '''
+            union
+                 (exceptions
+                 | where problemId != "ClientConnectionFailure at forward-request" and problemId != "ClientConnectionFailure at transfer-response"),
+                 (traces
+                 | where severityLevel >= 3 or (severityLevel >= 2 and customDimensions.SourceContext startswith "Digdir"))
+             | summarize Count = count()
+                 by
+                 Environment = tostring(customDimensions.AspNetCoreEnvironment),
+                 Type = itemType,
+                 problemId,
+                 message,
+                 severityLevel
+             | extend SeverityLevel = case(
+                                          severityLevel == 0, "Verbose",
+                                          severityLevel == 1, "Information",
+                                          severityLevel == 2, "Warning",
+                                          severityLevel == 3, "Error",
+                                          severityLevel == 4, "Critical",
+                                          tostring(severityLevel)
+                                      )
+             | extend Details = coalesce(message, problemId)
```

```suggestion
             | extend Details = coalesce(message, problemId)
             | extend Details = replace_string(Details, "Digdir.Domain.Dialogporten.", "")
```

Can we try something like this? We basically know the namespace is? 
Saves a few char. for the Slack message

---

Love it, tested and working fine, so I'll commit.